app/services/albums.js

```diff
@@ -0,0 +1,8 @@
+const fetch = require('node-fetch');
+
+exports.getListAlbums = (req, res, next) => {
+  return fetch('https://jsonplaceholder.typicode.com/todos/')
```

Comment:
- Url as config variable
- Use this endpoint `https://jsonplaceholder.typicode.com/albums/`